Example/Tests/JSZVCRRecorderTestCase.m

```diff
@@ -60,4 +60,20 @@ - (void)testRecordingNetworkCall {
     [self performSimpleVerifiedNetworkCall:nil];
 }
 
+- (void)testRecordingLargeChunkedNetworkCallRecordsAllData {
+    // Stubbing tests until I figure out a way to record on iOS 7 or 9
+    if (![[[UIDevice currentDevice] systemVersion] hasPrefix:@"8"]) {
+        return;
+    }
+    NSInteger numBytes = 30000;
+    [self performLargeChunked: numBytes verifiedNetworkCall:nil];
+    
+    NSArray *allRecordingsAtEndOfRun = [[JSZVCRRecorder sharedInstance].allRecordingsForPlist copy];
```

Comment:
Pre this PR, the world outside VCR saw the full data, but the *recording* got truncated. So as well as the tests that confirm that the data comes through as expected, we need to explicitly check the recording we're constructing.